tests/tck/features/MatchingSelfRelationships.feature

```diff
@@ -63,6 +63,7 @@ Feature: MatchingSelfRelationships
       | 1        |
     And no side effects
 
+@skip
```

Comment:
did it pass by accident?

Reply:
Yes. Bidirectional traversals typically return each edge twice (incoming and outgoing), but self-loops in Cypher still should only return once - I'm guessing this is a consequence of the edge uniqueness constraint we don't currently adhere to. I conferred with Roi and we agreed that these should be skips for the moment.